classes/PaymentModule.php

```diff
@@ -898,10 +898,6 @@ public static function getInstalledPaymentModules()
 
     public static function preCall($module_name)
     {
-        if (!parent::preCall($module_name)) {
-            return false;
-        }
```

Comment:
Seems like it was for PS 1.6 https://github.com/PrestaShop/PrestaShop-1.6/blob/master/classes/module/Module.php#L1882